pagerduty/resource_pagerduty_extension_test.go

```diff
@@ -68,10 +68,12 @@ func TestAccPagerDutyExtension_Basic(t *testing.T) {
 						"pagerduty_extension.foo", "extension_schema", "PJFWPEP"),
 					resource.TestCheckResourceAttr(
 						"pagerduty_extension.foo", "endpoint_url", url),
+					resource.TestCheckResourceAttr(
+						"pagerduty_extension.foo", "config", "{\n\t\"restrict\": \"any\",\n\t\"notify_types\": {\n\t\t\t\"resolve\": false,\n\t\t\t\"acknowledge\": false,\n\t\t\t\"assignments\": false\n\t}\n}\n"),
```

Comment:
I believe this should be:

```
"pagerduty_extension.foo", "config", "{\"notify_types\":{\"acknowledge\":false,\"assignments\":false,\"resolve\":false},\"restrict\":\"any\"}"),
```

Reply:
Changing this makes the acceptance to fail:

```
# make testacc TEST=./pagerduty/ TESTARGS='-run=TestAccPagerDutyExtension_Basic'
==> Checking that code complies with gofmt requirements...
TF_ACC=1 go test ./pagerduty/ -v -run=TestAccPagerDutyExtension_Basic -timeout 120m
=== RUN   TestAccPagerDutyExtension_Basic
--- FAIL: TestAccPagerDutyExtension_Basic (10.67s)
        testing.go:459: Step 0 error: Check failed: Check 5/5 error: pagerduty_extension.foo: Attribute 'config' expected "{\"restrict\": \"any\",\"notify_types\": {\"resolve\": false,\"acknowledge\": false,\"assignments\": false}}", got "{\n\t\"restrict\": \"any\",\n\t\"notify_types\": {\n\t\t\t\"resolve\": false,\n\t\t\t\"acknowledge\": false,\n\t\t\t\"assignments\": false\n\t}\n}\n"
FAIL
FAIL    github.com/terraform-providers/terraform-provider-pagerduty/pagerduty   10.674s
GNUmakefile:17: recipe for target 'testacc' failed
make: *** [testacc] Error 1
```

Reply:
I think that's due to the `d.Set` issue above, 
because we're not actually reading back the config from the API.